docs/wcs/note_sip.rst

```diff
@@ -0,0 +1,85 @@
+.. include:: references.rst
+.. doctest-skip-all
+.. _note_sip:
```

Comment:
I think there's an ``:orphan:`` directive or similar if the page isn't included in a TOC (otherwise I think sphinx may emit a warning)